### PR TITLE
LITE-29259 Create credential data model

### PR DESCRIPTION
**Pull request by jonatrios**

### CODE REVIEW

File: connect_bi_reporter/credentials/models.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+
+class Credential(Model):
+    __tablename__ = "credential"
+
+    PREFIX = 'CRED'
+
+    id = db.Column(db.String(20), primary_key=True)
+    name = db.Column(db.String(64))
+    connection_string = db.Column(db.String(256))
```
Comment (maxipavlovic) [inline connect_bi_reporter/credentials/models.py:15]: For now I am fine, just curious:
shall connection be the string always? Or maybe some structured field is needed instead or in addition?

File: connect_bi_reporter/credentials/models.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+from connect_bi_reporter.db import Model
+
+
+class Credential(Model):
+    __tablename__ = "credential"
+
+    PREFIX = 'CRED'
+
+    id = db.Column(db.String(20), primary_key=True)
```
Comment (maxipavlovic) [inline connect_bi_reporter/credentials/models.py:13]: What about name / account_id (something else) being non-nullable by default?

**Review by maxipavlovic — COMMENTED** [review]
In general LGTM.

File: connect_bi_reporter/credentials/models.py (hunk 1 above)
Comment (jonatrios) [reply connect_bi_reporter/credentials/models.py:15]: Is true that connection can be constructed base on account_name and account_key, but from azure interface user can copy and paste it directly to the extension as "connection string", so account and key will be already included:
image

File: connect_bi_reporter/credentials/models.py (hunk 2 above)
Comment (jonatrios) [reply connect_bi_reporter/credentials/models.py:13]: updated!

**Review by akodelia — APPROVED** [review]

**Pull request merged**